### PR TITLE
TIED-59: Replace PyXB with lxml

### PR DESCRIPTION
**Pull request by danipran**
## Description

Replace PyXB with lxml by rewriting the old JHS exporter using lxml. 

## Related Issue(s)

**[TIED-59](https://helsinkisolutionoffice.atlassian.net/browse/TIED-59)**

## Motivation and Context

PyXB acts a a blocker for Python and package version updates; it is no longer maintained, and will stop functioning in Python >=3.8. Python 3.7 LTS ends this year, and some of the packages have already dropped 3.7 support.

There were some more recent forks for PyXB, but they did not seem to have any regular updates, which is why we ended up with the rewrite solution.

## How Has This Been Tested?

Unit tests.

## Additional Notes

- While this solution generates an almost identical document compared to the old one (just a larger indentation), there's no guarantee that this works in production. 
- This PR also splits the code quite a lot more than the original implementation and restructures it, moving all the relevant code under the `metarecord.exporter.jhs` package.
- No automatic binding generation anymore. If there are any changes to the schema, they need to be done manually.
- Please tell if there's a better way of doing things with lxml; I had trouble with attribute namespaces at least and ended up doing a wrapper for the elements so I don't have to repeat myself too much.

[TIED-59]: https://helsinkisolutionoffice.atlassian.net/browse/TIED-59?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: metarecord/exporter/jhs/builder.py [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+                kieliKoodi="fi",
+            )
+        ),
+        bindings.YHTEYSHENKILO_NIMI("Tiedonhallinta"),
+        bindings.TOS_VERSIO(TOS_VERSION),
+        bindings.TILA_KOODI("3"),
+        bindings.ORGANISAATIO_NIMI("Helsingin kaupunki"),
+        bindings.LISATIEDOT(
+            "JHS 191 XML {:%Y-%m-%d %H:%M%Z} {}".format(
+                datetime.now(tz=pytz.timezone(settings.TIME_ZONE)),
```
Comment (voneiden) [inline metarecord/exporter/jhs/builder.py:238]: Technically not in scope of the PR but caught my eye

This approach appears to be unique, which is mildly surprising. [https://www.google.com/search?q="datetime.now(tz=pytz.timezone(settings.TIME_ZONE))"](https://www.google.com/search?client=firefox-b-d&q=%22datetime.now%28tz%3Dpytz.timezone%28settings.TIME_ZONE%29%29%22)

I think it works OK.. at least with Europe/Helsinki, but standard approach would be to use django's `timezone.localtime(timezone.now())`

**Review by voneiden — CHANGES_REQUESTED** [review]

File: metarecord/exporter/jhs/builder.py (hunk 1 above)
Comment (danipran) [reply metarecord/exporter/jhs/builder.py:238]: This is the way. Will fix it.

File: metarecord/exporter/jhs/builder.py [hunk 2]
```diff
@@ -0,0 +1,255 @@
+import uuid
```
[the remark this answers is not in the document]
Comment (danipran) [reply metarecord/exporter/jhs/builder.py:1]: Yeah, sure.

**Comment by danipran** [comment]
Will do some rebasing and squash the commits, in addition to adding @voneiden's changes.

**Review by voneiden — APPROVED** [review]

**Pull request merged**